packages/@react-aria/utils/src/isVirtualEvent.ts

```diff
@@ -46,8 +46,12 @@ export function isVirtualPointerEvent(event: PointerEvent) {
   // Cannot use "event.pressure === 0" as the sole check due to Safari pointer events always returning pressure === 0
   // instead of .5, see https://bugs.webkit.org/show_bug.cgi?id=206216. event.pointerType === 'mouse' is to distingush
   // Talkback double tap from Windows Firefox touch screen press
+
+  console.log(!isAndroid());
+  console.log(event.width === 0 && event.height === 0);
+
   return (
-    (event.width === 0 && event.height === 0) ||
+    (!isAndroid() && event.width === 0 && event.height === 0) ||
```

Comment:
Did some more testing, seems like Talkback will issue a event with `width: 0` and `height: 0` when double tapping on things like a TableView's cell and a ListView's row's text context (not the cell, but the actual text) and as such will return `false` for this `isVirtualPointerEvent` check with the changes made in this PR. I haven't found a case where this seems to cause any problems yet though... Annoyingly enough, those `width: 0 + height: 0` events also have `e.pointerType: 'touch'`, making it difficult to distinguish them from valid touch events that happen to have `width: 0 + height: 0`

Reply:
I just remembered this bug I filed against Chrome, would be good to get some stars on it so we'd have a better way of distinguishing a virtual click event from TalkBack: https://bugs.chromium.org/p/chromium/issues/detail?id=1271444

Reply:
Hmm that doesn't seem great even if it isn't necessarily causing problems right now. We could change the check to `isIOS() && event.width===0 && event.height===0` so long as `width: 0 + height: 0` are exclusive to it. We would need to update https://github.com/adobe/react-spectrum/blob/8987f9283871b1bda0acc4637096966275f82167/packages/%40react-aria/interactions/test/usePress.test.js#L536 so that iOS is mocked otherwise it fails.

Reply:
Did some more testing, still haven't found a scenario where the cases where Talkback fires a event with `width: 0` and `height: 0` is problematic in our components. Lets just go with what you have for now.

Reply:
Okay, sounds good. I'll mark it as ready for review